### ISSUE TITLE
button icons are not hidden in rename chat thread field

### ISSUE THREAD

**Issue by derekjackson-das** [danny-avila/LibreChat#3828]
**Issue Description**:
The icons within the buttons used to rename a chat thread are not hidden from assistive technologies. As a result, screen readers announce the icons as content. The icons should be hidden to improve accessibility.

Related to #3826, #3825 

**WCAG Criteria:**
[1.1.1 Non-text Content - A](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)

**Screenshot or Screen Recording**:
[Screen Reader Review](https://share.zight.com/eDuZolvL)

<img width="2021" alt="Screenshot 2024-08-28 at 4 21 17 PM" src="https://github.com/user-attachments/assets/401b4268-8452-44c3-a77b-2454029eec83">

**Snippet**:

```
<div class="flex gap-1">
  <button>
    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-x transition-color h-4 w-4 duration-200 ease-in-out hover:opacity-70">
      <path d="M18 6 6 18"></path>
      <path d="m6 6 12 12"></path>
    </svg>
  </button>
  <button>
    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="lucide lucide-check transition-color h-4 w-4 duration-200 ease-in-out hover:opacity-70">
      <path d="M20 6 9 17l-5-5"></path>
    </svg>
  </button>
</div>
```
**Notes**:
Add the aria-hidden attribute to the svg tags with a value of "true". This ensures the icons are hidden from screen readers, as they do not provide important information, like the name of the button. In this case the buttons should have an accessible name added as well, see #3826
